symfony/plugins/orangehrmPimPlugin/entity/Employee.php

```diff
@@ -458,10 +458,7 @@ class Employee
     /**
      * @var Collection
      *
-     * @ORM\ManyToMany(targetEntity="OrangeHRM\Entity\EmployeeMembership", mappedBy="Employee")
-     * @ORM\JoinColumns({
-     *   @ORM\JoinColumn(name="emp_number", referencedColumnName="emp_number")
-     * })
+     * @ORM\OneToMany (targetEntity="OrangeHRM\Entity\EmployeeMembership", mappedBy="employee")
```

Comment:
should include `@ORM\JoinColumn(name="emp_number", referencedColumnName="emp_number")`